packages/core/admin/utils/get-plugins-path.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const { join, resolve } = require('path');
+const fs = require('fs-extra');
+// eslint-disable-next-line node/no-extraneous-require
+const glob = require('glob');
+
+const getPluginsPath = () => {
+  const rootPath = join('..', '..', '..', 'packages');
+  const corePath = join(rootPath, 'core', '*');
+  const pluginsPath = join(rootPath, 'plugins', '*');
+  const corePackageDirs = glob.sync(corePath);
+  const pluginsPackageDirs = glob.sync(pluginsPath);
+  const packageDirs = [...corePackageDirs, ...pluginsPackageDirs].filter(dir => {
+    const isCoreAdmin = dir.includes('packages/core/admin/');
+    const pathToEntryPoint = join(dir, 'admin', 'src', 'index.js');
+    const doesAdminFolderExist = fs.pathExistsSync(pathToEntryPoint);
+
+    return !isCoreAdmin && doesAdminFolderExist;
+  });
+
+  return packageDirs.map(dir => resolve(__dirname, '..', join(dir, 'admin', 'src')));
+};
+
+module.exports = getPluginsPath;
```

Comment:
Could you add a test for this file? :) It's nicely written, but a nightmare to refactor later.